packages/telemetry/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@astrojs/telemetry",
   "version": "3.0.1",
   "type": "module",
-  "types": "./dist/types/index.d.ts",
+  "types": "./dist/index.d.ts",
```

Comment:
I'm not sure why the telemetry package built itself this way, but it was the only package doing so. I changed it to work like all the other packages.